.gitignore

```diff
@@ -16,6 +16,8 @@ resources/dbs/sequencer-db/
 resources/dbs/full-node-db/
 resources/dbs/batch-prover-db/
 resources/dbs/light-client-prover-db/
+data/
```

Comment:
Anything we write to disk should go into `resources`. We should keep the root clean of dynamically created files / directories.

Reply:
I also searched where we use this directory and could not find any usage of this.

Reply:
Yeah my bad, it is something else will remove